server/client_manager.py

```diff
@@ -118,16 +119,30 @@ def send_command(self, command: str, *args):
 
             Args:
                 command (str): command name
-                *args: list of arguments
+                *args: tuple containing the packet arguments
             """
             if args:
                 if command == 'MS':
                     for evi_num in range(len(self.evi_list)):
                         if self.evi_list[evi_num] == args[11]:
-                            lst = list(args)
+                            lst = list(args) # convert to a list so we can modify it
                             lst[11] = evi_num
-                            args = tuple(lst)
+                            args = tuple(lst) # then convert back to a tuple when finished
                             break
+                    # <2.9 can't parse Y offset so we strip it out based on version
+                    c_version = self.version.split('.') # i.e. ['2','9','0'] or ['2','8','5']
+                    if len(c_version) > 1: # sanity check, version strings /should/ be in x.x.x format 
+                        try:
```

Comment:
What's the need to try except this?
Seems like the lazy mans way of handling things. Maybe do some kind of validation for version?

Reply:
version strings aren't validated because they can be _anything._ the try/except block is there because if someone sets their version string to "okay. this is epic" that's obviously not giving us any useful information so there's no point in trying to figure out whether their client supports y-offset

Reply:
added validation